src/settings/utils/time.js

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/prefer-default-export
+export const OFFSET_HOURS = (new Date().getTimezoneOffset()) / 60;
```

Comment:
Are you confident this does the right thing? `new Date()` will create a timestamp in the _browser's_ timezone. Shouldn't we be making this calculation in the _tenant's_ timezone?

Reply:
Hello @zburke  Current bug reproduceable only when browser's timezone is less then 0. This is problem with "react-big-calendar" all the days shifts over by 1 and this is workaround for resolve problem with "react-big-calendar".

Reply:
OK. If the problem is specific to the browser's timezone, then this adjustment makes sense.